src/olympia/lib/cache.py

```diff
@@ -12,28 +12,41 @@
 from django.conf import settings
 
 
-def make_key(key=None, with_locale=True):
+def make_key(key, with_locale=True, normalize=False):
```

Comment:
Is this the same as the `make_key` function mentioned in https://docs.djangoproject.com/en/1.11/topics/cache/#cache-key-transformation?  If so, seems best to make the signature match now so we can drop it in.  (If not, maybe a different name to avoid confusion)